frontend/imports/utils/conversion.js

```diff
@@ -1,15 +1,36 @@
-import { Dapple, web3 } from 'meteor/makerotc:dapple';
+import { Dapple } from 'meteor/makerotc:dapple';
 import { BigNumber } from 'meteor/ethereum:web3';
 
-export default function convertToWei(amount, token) {
-  const tokenSpecs = Dapple.getTokenSpecs(token);
-  const precision = tokenSpecs !== undefined ? tokenSpecs.precision : 18;
-  if (precision === 18) {
-    return web3.toWei(amount);
+export function convertToAbsolute(amount, token) {
```

Comment:
I think this function might be misnamed? I keep thinking it's referring to absolute value (e.g. `|-17| = 17` ) but it seems to be transforming according to the token's precision